vimrc

```diff
@@ -227,7 +227,18 @@ map <silent> <LocalLeader>nr :NERDTree<CR>
 map <silent> <LocalLeader>nf :NERDTreeFind<CR>
 
 " FZF
-map <silent> <leader>ff :Files<CR>
+function! SmartFuzzy()
+  let root = split(system('git rev-parse --show-toplevel'), '\n')
+  echo "HERE"
```

Comment:
Is this intentional?

Reply:
@hongtron nice catch. Debugging is hard.